web/README.md

```diff
@@ -11,8 +11,9 @@ Prerequisites:
 - Ensure you have `NodeJS` and `pnpm` installed locally (`brew install pnpm`)
 - Run `pnpm install` in both the web and mockserver directories
 
-1. Start the mockserver: `pnpm run mockserver`
-2. Run app in another tab: `pnpm dev`
+1. Install mockserver dependencies `cd ../mockserver; pnpm install; cd ../web`
```

Comment:
This is already included in the readme directly above. I don't think this is necessary.

Reply:
I completely missed it. I'll close this PR.

Reply:
All good! Thanks for taking the time to create the PR regardless!